// components/General/Accordion/AccordionMob.js
import { styled } from "@mui/material/styles";
import MuiAccordion from "@mui/material/Accordion";
import MuiAccordionSummary from "@mui/material/AccordionSummary";
import MuiAccordionDetails from "@mui/material/AccordionDetails";

export const Accordion = styled((props) => (
  <MuiAccordion disableGutters square {...props} />
))(({ theme }) => ({
  "& .MuiButtonBase-root.Mui-expanded": {
    boxShadow: "0px 0px 8px rgba(0, 0, 0, 0.15)",
    borderRadius: "8px",
  },
  boxShadow: `0px 0px 8px rgba(0, 0, 0, 0.15)`,
  borderRadius: `8px`,
  marginBottom: `4px`,
  "&:before": {
    opacity: 0,
  },
  "& .MuiAccordion-root.Mui-expanded": {
    margin: "4px 0",
  },
  "& .MuiPaper-root.Mui-expanded": {
    margin: 0,
  },
}));

export const AccordionSummary = styled((props) => (
  <MuiAccordionSummary expandIcon={"–"} {...props} />
))(({ theme }) => ({
  padding: `0 25px 0 30px`,
  "& .MuiTypography-root": {
    fontWeight: "bold",
    fontSize: "20px",
    lineHeight: "23px",
    color: "#4e4e4e",
    margin: "8px 0",
  },
  "& .MuiAccordionSummary-content.Mui-expanded": {
    margin: "12px 0",
  },
  "& .MuiAccordionSummary-expandIconWrapper": {
    fontWeight: "bold",
    fontSize: "36px",
    lineHeight: "42px",
    color: "#8b75c8",
  },
  "& .MuiAccordionSummary-expandIconWrapper::before": {
    content: "'–'",
    position: "absolute",
    top: "48%",
    left: "49%",
    transform: "translate(-50%, -50%) rotate(90deg)",
    opacity: 1,
  },
  "& .MuiAccordionSummary-expandIconWrapper.Mui-expanded::before": {
    opacity: 0,
  },
}));

export const AccordionDetails = styled((props) => (
  <MuiAccordionDetails {...props} />
))(({ theme }) => ({
  padding: `26px 22px 29px`,
  "& .MuiTypography-root": {
    fontWeight: "normal",
    fontSize: "16px",
    lineHeight: "30px",
    color: "#494949",
  },
  "& .MuiTypography-root:not(:last-child)": {
    marginBottom: "10px",
  },
}));

export default function CustomizedAccordions(props) {
  return <div>{props.children}</div>;
}
