// Consts/catalog/catalog.js
import Photo1 from "img/photo1.png";
import Photo2 from "img/baseus-d01-encok.png";
import Photo3 from "img/photo3.png";
import Photo4 from "img/m1.png";
import Photo5 from "img/m2.jpeg";
import Photo6 from "img/m3.png";
import Photo61 from "img/m31.jpeg";
import Photo62 from "img/m32.jpeg";
import Photo63 from "img/m33.jpeg";
import Photo64 from "img/m34.webp";
import Photo65 from "img/m35.webp";
import Photo66 from "img/m36.jpeg";
import Photo7 from "img/m4.jpeg";
import Photo8 from "img/m5.jpeg";
import Photo9 from "img/m6.jpeg";
import Photo10 from "img/m7.jpeg";
import Photo11 from "img/m8.jpeg";

export const CATALOG = [
  {
    price: "1950",
    lastPrice: "2250",
    title: "Наушники Bluetooth Baseus Encok D01 NGD01-09 (red)",
    rate: 5,
    currency: "RUB",
    image: [Photo1],
    id: 1,
    vendorCode: 736592,
    tags: ["apple", "sale"],
    description: [
      "Комфортные и стильные беспроводные наушники Awei A500BL оснащены двумя 40-ка миллиметровыми динамиками, обеспечивающими качественное звучание высоких и низких частот с большим запасом громкости.",
      "Подключитесь по Bluetooth к своему смартфону и наслаждайтесь музыкой с беспроводной гарнитурой Awei A500BL , чистое стерео-звучание, мощные басы, она отлично подойдет во время занятий спортом, на пробежке.",
      "Удобные амбушюры плотно закрывают уши и изолируют внешние шумы, тем самым делая прослушивание музыки более комфортным и приятным. С наушниками Awei A500BL вы сможете наслаждаться любимыми песнями, не обременяя себя неудобными проводами.",
    ],
    link: "./product",
  },
  {
    price: "2950",
    lastPrice: "3250",
    title: "Наушники Bluetooth Baseus Encok D01 NGD01-09 (red)",
    rate: 5,
    currency: "RUB",
    image: [Photo2],
    id: 2,
    vendorCode: 756322,
    tags: ["apple"],
    description: [
      "Комфортные и стильные беспроводные наушники Awei A500BL оснащены двумя 40-ка миллиметровыми динамиками, обеспечивающими качественное звучание высоких и низких частот с большим запасом громкости.",
      "Подключитесь по Bluetooth к своему смартфону и наслаждайтесь музыкой с беспроводной гарнитурой Awei A500BL , чистое стерео-звучание, мощные басы, она отлично подойдет во время занятий спортом, на пробежке.",
      "Удобные амбушюры плотно закрывают уши и изолируют внешние шумы, тем самым делая прослушивание музыки более комфортным и приятным. С наушниками Awei A500BL вы сможете наслаждаться любимыми песнями, не обременяя себя неудобными проводами.",
    ],
    link: "./product",
  },
  {
    price: "",
    title: "",
    rate: 0,
    currency: "",
    image: "",
    id: 3,
    isBanner: true,
    tags: [""],
  },
  {
    price: "1750",
    lastPrice: "2050",
    title: "Наушники Bluetooth Baseus Encok D01 NGD01-09 (red)",
    rate: 3,
    currency: "RUB",
    image: [Photo3],
    id: 4,
    vendorCode: 840265,
    tags: ["samsung"],
    description: [
      "Комфортные и стильные беспроводные наушники Awei A500BL оснащены двумя 40-ка миллиметровыми динамиками, обеспечивающими качественное звучание высоких и низких частот с большим запасом громкости.",
      "Подключитесь по Bluetooth к своему смартфону и наслаждайтесь музыкой с беспроводной гарнитурой Awei A500BL , чистое стерео-звучание, мощные басы, она отлично подойдет во время занятий спортом, на пробежке.",
      "Удобные амбушюры плотно закрывают уши и изолируют внешние шумы, тем самым делая прослушивание музыки более комфортным и приятным. С наушниками Awei A500BL вы сможете наслаждаться любимыми песнями, не обременяя себя неудобными проводами.",
    ],
    link: "./product",
  },
  {
    price: "1450",
    lastPrice: "1750",
    title: "Наушники Bluetooth Baseus Encok D01 NGD01-09 (red)",
    rate: 3,
    currency: "RUB",
    image: [Photo4],
    id: 5,
    vendorCode: 857365,
    tags: ["samsung", "sale"],
    description: [
      "Комфортные и стильные беспроводные наушники Awei A500BL оснащены двумя 40-ка миллиметровыми динамиками, обеспечивающими качественное звучание высоких и низких частот с большим запасом громкости.",
      "Подключитесь по Bluetooth к своему смартфону и наслаждайтесь музыкой с беспроводной гарнитурой Awei A500BL , чистое стерео-звучание, мощные басы, она отлично подойдет во время занятий спортом, на пробежке.",
      "Удобные амбушюры плотно закрывают уши и изолируют внешние шумы, тем самым делая прослушивание музыки более комфортным и приятным. С наушниками Awei A500BL вы сможете наслаждаться любимыми песнями, не обременяя себя неудобными проводами.",
    ],
    link: "./product",
  },
  {
    price: "2350",
    lastPrice: "2650",
    title: "Наушники Bluetooth Baseus Encok D01 NGD01-09 (red)",
    rate: 5,
    currency: "RUB",
    image: [Photo5],
    id: 6,
    vendorCode: 174058,
    tags: ["apple", "samsung"],
    description: [
      "Комфортные и стильные беспроводные наушники Awei A500BL оснащены двумя 40-ка миллиметровыми динамиками, обеспечивающими качественное звучание высоких и низких частот с большим запасом громкости.",
      "Подключитесь по Bluetooth к своему смартфону и наслаждайтесь музыкой с беспроводной гарнитурой Awei A500BL , чистое стерео-звучание, мощные басы, она отлично подойдет во время занятий спортом, на пробежке.",
      "Удобные амбушюры плотно закрывают уши и изолируют внешние шумы, тем самым делая прослушивание музыки более комфортным и приятным. С наушниками Awei A500BL вы сможете наслаждаться любимыми песнями, не обременяя себя неудобными проводами.",
    ],
    link: "./product",
  },
  {
    price: "2000",
    lastPrice: "2250",
    title: "Наушники Bluetooth Baseus Encok D01 NGD01-09 (red)",
    rate: 4.8,
    currency: "RUB",
    image: [Photo6, Photo61, Photo62, Photo63, Photo64, Photo65, Photo66],
    id: 7,
    vendorCode: 273040,
    tags: ["apple", "xiaomi"],
    characteristics: [
      { key: "Bluetooth", value: "V4.1 CSR" },
      { key: "Профили Bluetooth", value: "A2DP, AVRCP, HSP, HFP" },
      { key: "Чувствительность", value: "116 дБ" },
      { key: "Диапазон воспроизводимых частот", value: "20Hz-20kHz" },
      { key: "Динамик (диффузор)", value: "40мм" },
      { key: "Импеданс", value: "16 Ом" },
      { key: "Радиус действия", value: "10 метров" },
      { key: "В режиме ожидания", value: "больше 200 часов" },
      { key: "В режиме разговора", value: "3 часа" },
      { key: "В режиме воспроизведения", value: "12 часов" },
      { key: "Время заряда от USB", value: "около 3 часов" },
      { key: "Вес наушников", value: "190 граммов" },
    ],
    description: [
      "Комфортные и стильные беспроводные наушники Awei A500BL оснащены двумя 40-ка миллиметровыми динамиками, обеспечивающими качественное звучание высоких и низких частот с большим запасом громкости.",
      "Подключитесь по Bluetooth к своему смартфону и наслаждайтесь музыкой с беспроводной гарнитурой Awei A500BL , чистое стерео-звучание, мощные басы, она отлично подойдет во время занятий спортом, на пробежке.",
      "Удобные амбушюры плотно закрывают уши и изолируют внешние шумы, тем самым делая прослушивание музыки более комфортным и приятным. С наушниками Awei A500BL вы сможете наслаждаться любимыми песнями, не обременяя себя неудобными проводами.",
    ],
    link: "./product",
  },
  {
    price: "1900",
    lastPrice: "2250",
    title: "Наушники Bluetooth Baseus Encok D01 NGD01-09 (red)",
    rate: 4,
    currency: "RUB",
    image: [Photo7],
    id: 8,
    vendorCode: 112845,
    tags: ["xiaomi", "sale"],
    description: [
      "Комфортные и стильные беспроводные наушники Awei A500BL оснащены двумя 40-ка миллиметровыми динамиками, обеспечивающими качественное звучание высоких и низких частот с большим запасом громкости.",
      "Подключитесь по Bluetooth к своему смартфону и наслаждайтесь музыкой с беспроводной гарнитурой Awei A500BL , чистое стерео-звучание, мощные басы, она отлично подойдет во время занятий спортом, на пробежке.",
      "Удобные амбушюры плотно закрывают уши и изолируют внешние шумы, тем самым делая прослушивание музыки более комфортным и приятным. С наушниками Awei A500BL вы сможете наслаждаться любимыми песнями, не обременяя себя неудобными проводами.",
    ],
    link: "./product",
  },
  {
    price: "3450",
    lastPrice: "4000",
    title: "Наушники Bluetooth Baseus Encok D01 NGD01-09 (red)",
    rate: 5,
    currency: "RUB",
    image: [Photo8],
    id: 9,
    vendorCode: 263859,
    tags: ["apple", "xiaomi", "huawei", "samsung"],
    description: [
      "Комфортные и стильные беспроводные наушники Awei A500BL оснащены двумя 40-ка миллиметровыми динамиками, обеспечивающими качественное звучание высоких и низких частот с большим запасом громкости.",
      "Подключитесь по Bluetooth к своему смартфону и наслаждайтесь музыкой с беспроводной гарнитурой Awei A500BL , чистое стерео-звучание, мощные басы, она отлично подойдет во время занятий спортом, на пробежке.",
      "Удобные амбушюры плотно закрывают уши и изолируют внешние шумы, тем самым делая прослушивание музыки более комфортным и приятным. С наушниками Awei A500BL вы сможете наслаждаться любимыми песнями, не обременяя себя неудобными проводами.",
    ],
    link: "./product",
  },
  {
    price: "2700",
    lastPrice: "3150",
    title: "Наушники Bluetooth Baseus Encok D01 NGD01-09 (red)",
    rate: 5,
    currency: "RUB",
    image: [Photo9],
    id: 10,
    vendorCode: 957385,
    tags: ["huawei"],
    description: [
      "Комфортные и стильные беспроводные наушники Awei A500BL оснащены двумя 40-ка миллиметровыми динамиками, обеспечивающими качественное звучание высоких и низких частот с большим запасом громкости.",
      "Подключитесь по Bluetooth к своему смартфону и наслаждайтесь музыкой с беспроводной гарнитурой Awei A500BL , чистое стерео-звучание, мощные басы, она отлично подойдет во время занятий спортом, на пробежке.",
      "Удобные амбушюры плотно закрывают уши и изолируют внешние шумы, тем самым делая прослушивание музыки более комфортным и приятным. С наушниками Awei A500BL вы сможете наслаждаться любимыми песнями, не обременяя себя неудобными проводами.",
    ],
    link: "./product",
  },
  {
    price: "1340",
    lastPrice: "1950",
    title: "Наушники Bluetooth Baseus Encok D01 NGD01-09 (red)",
    rate: 3,
    currency: "RUB",
    image: [Photo10],
    id: 11,
    vendorCode: 637495,
    tags: ["huawei"],
    description: [
      "Комфортные и стильные беспроводные наушники Awei A500BL оснащены двумя 40-ка миллиметровыми динамиками, обеспечивающими качественное звучание высоких и низких частот с большим запасом громкости.",
      "Подключитесь по Bluetooth к своему смартфону и наслаждайтесь музыкой с беспроводной гарнитурой Awei A500BL , чистое стерео-звучание, мощные басы, она отлично подойдет во время занятий спортом, на пробежке.",
      "Удобные амбушюры плотно закрывают уши и изолируют внешние шумы, тем самым делая прослушивание музыки более комфортным и приятным. С наушниками Awei A500BL вы сможете наслаждаться любимыми песнями, не обременяя себя неудобными проводами.",
    ],
    link: "./product",
  },
  {
    price: "1800",
    lastPrice: "2250",
    title: "Наушники Bluetooth Baseus Encok D01 NGD01-09 (red)",
    rate: 4,
    currency: "RUB",
    image: [Photo11],
    id: 12,
    vendorCode: 211934,
    tags: ["huawei"],
    description: [
      "Комфортные и стильные беспроводные наушники Awei A500BL оснащены двумя 40-ка миллиметровыми динамиками, обеспечивающими качественное звучание высоких и низких частот с большим запасом громкости.",
      "Подключитесь по Bluetooth к своему смартфону и наслаждайтесь музыкой с беспроводной гарнитурой Awei A500BL , чистое стерео-звучание, мощные басы, она отлично подойдет во время занятий спортом, на пробежке.",
      "Удобные амбушюры плотно закрывают уши и изолируют внешние шумы, тем самым делая прослушивание музыки более комфортным и приятным. С наушниками Awei A500BL вы сможете наслаждаться любимыми песнями, не обременяя себя неудобными проводами.",
    ],
    link: "./product",
  },
  {
    price: "1950",
    lastPrice: "2250",
    title: "Наушники Bluetooth Baseus Encok D01 NGD01-09 (red)",
    rate: 5,
    currency: "RUB",
    image: [Photo1],
    id: 13,
    vendorCode: 746393,
    tags: ["apple", "sale"],
    description: [
      "Комфортные и стильные беспроводные наушники Awei A500BL оснащены двумя 40-ка миллиметровыми динамиками, обеспечивающими качественное звучание высоких и низких частот с большим запасом громкости.",
      "Подключитесь по Bluetooth к своему смартфону и наслаждайтесь музыкой с беспроводной гарнитурой Awei A500BL , чистое стерео-звучание, мощные басы, она отлично подойдет во время занятий спортом, на пробежке.",
      "Удобные амбушюры плотно закрывают уши и изолируют внешние шумы, тем самым делая прослушивание музыки более комфортным и приятным. С наушниками Awei A500BL вы сможете наслаждаться любимыми песнями, не обременяя себя неудобными проводами.",
    ],
    link: "./product",
  },
  {
    price: "2950",
    lastPrice: "3450",
    title: "Наушники Bluetooth Baseus Encok D01 NGD01-09 (red)",
    rate: 5,
    currency: "RUB",
    image: [Photo2],
    id: 14,
    vendorCode: 589393,
    tags: ["apple"],
    description: [
      "Комфортные и стильные беспроводные наушники Awei A500BL оснащены двумя 40-ка миллиметровыми динамиками, обеспечивающими качественное звучание высоких и низких частот с большим запасом громкости.",
      "Подключитесь по Bluetooth к своему смартфону и наслаждайтесь музыкой с беспроводной гарнитурой Awei A500BL , чистое стерео-звучание, мощные басы, она отлично подойдет во время занятий спортом, на пробежке.",
      "Удобные амбушюры плотно закрывают уши и изолируют внешние шумы, тем самым делая прослушивание музыки более комфортным и приятным. С наушниками Awei A500BL вы сможете наслаждаться любимыми песнями, не обременяя себя неудобными проводами.",
    ],
    link: "./product",
  },
  {
    price: "1750",
    lastPrice: "2250",
    title: "Наушники Bluetooth Baseus Encok D01 NGD01-09 (red)",
    rate: 3,
    currency: "RUB",
    image: [Photo3],
    id: 15,
    vendorCode: 475592,
    tags: ["samsung"],
    description: [
      "Комфортные и стильные беспроводные наушники Awei A500BL оснащены двумя 40-ка миллиметровыми динамиками, обеспечивающими качественное звучание высоких и низких частот с большим запасом громкости.",
      "Подключитесь по Bluetooth к своему смартфону и наслаждайтесь музыкой с беспроводной гарнитурой Awei A500BL , чистое стерео-звучание, мощные басы, она отлично подойдет во время занятий спортом, на пробежке.",
      "Удобные амбушюры плотно закрывают уши и изолируют внешние шумы, тем самым делая прослушивание музыки более комфортным и приятным. С наушниками Awei A500BL вы сможете наслаждаться любимыми песнями, не обременяя себя неудобными проводами.",
    ],
    link: "./product",
  },
  {
    price: "2000",
    lastPrice: "2550",
    title: "Наушники Bluetooth Baseus Encok D01 NGD01-09 (red)",
    rate: 4,
    currency: "RUB",
    image: [Photo6],
    id: 16,
    vendorCode: 164657,
    tags: ["apple", "xiaomi"],
    description: [
      "Комфортные и стильные беспроводные наушники Awei A500BL оснащены двумя 40-ка миллиметровыми динамиками, обеспечивающими качественное звучание высоких и низких частот с большим запасом громкости.",
      "Подключитесь по Bluetooth к своему смартфону и наслаждайтесь музыкой с беспроводной гарнитурой Awei A500BL , чистое стерео-звучание, мощные басы, она отлично подойдет во время занятий спортом, на пробежке.",
      "Удобные амбушюры плотно закрывают уши и изолируют внешние шумы, тем самым делая прослушивание музыки более комфортным и приятным. С наушниками Awei A500BL вы сможете наслаждаться любимыми песнями, не обременяя себя неудобными проводами.",
    ],
    link: "./product",
  },
  {
    price: "1900",
    lastPrice: "2250",
    title: "Наушники Bluetooth Baseus Encok D01 NGD01-09 (red)",
    rate: 4,
    currency: "RUB",
    image: [Photo7],
    id: 17,
    vendorCode: 747753,
    tags: ["xiaomi", "sale"],
    description: [
      "Комфортные и стильные беспроводные наушники Awei A500BL оснащены двумя 40-ка миллиметровыми динамиками, обеспечивающими качественное звучание высоких и низких частот с большим запасом громкости.",
      "Подключитесь по Bluetooth к своему смартфону и наслаждайтесь музыкой с беспроводной гарнитурой Awei A500BL , чистое стерео-звучание, мощные басы, она отлично подойдет во время занятий спортом, на пробежке.",
      "Удобные амбушюры плотно закрывают уши и изолируют внешние шумы, тем самым делая прослушивание музыки более комфортным и приятным. С наушниками Awei A500BL вы сможете наслаждаться любимыми песнями, не обременяя себя неудобными проводами.",
    ],
    link: "./product",
  },
];

export default CATALOG;
